### PR TITLE
Feature/rh 16 UI for user data editing

### LINKED ISSUE #16: UI for data editing
As a user I want to edit my own data to make it accessible for other users, these changes should be reflected in the database (Team APMW).
@

- [x] Inputfelder
- [x] Telefonnummer
- [x] Email
- [x] Raum
- [x] Office-Hour
- [x] Name (mit Polze absprechen, in welchem Format, OpenId beachten)
- [x] Daten aus OpenID übernehmen (E-Mail und voller Name)
- [x] Aktuelle Daten aus Datenbank übernehmen
- [x] Speicherbutton
- [x] Beim Speichern werden die Änderungen übernommen und man kommt zur Show-Seite
- [x] Erormessages anzeigen

### PR DESCRIPTION
**Pull request by BenediktV**
closes #16

### CODE REVIEW

File: app/controllers/openingtimes_controller.rb [hunk 1]
```diff
@@ -48,10 +51,10 @@ def update
   # DELETE /openingtimes/1 or /openingtimes/1.json
   def destroy
     @openingtime.destroy
-    respond_to do |format|
-      format.html { redirect_to openingtimes_url, notice: "Openingtime was successfully destroyed." }
-      format.json { head :no_content }
-    end
+    #     respond_to do |format|
+    #       format.html { redirect_to openingtimes_url, notice: "Openingtime was successfully destroyed." }
+    #       format.json { head :no_content }
```
Comment (TheGrayStone) [inline app/controllers/openingtimes_controller.rb:56]: Why is this commented out? Do we still need it somewhere down the line or can this be removed for good?

File: app/javascript/packs/application.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
 
+// See https://rubyyagi.com/how-to-use-bootstrap-and-jquery-in-rails-6-with-webpacker/
```
Comment (TheGrayStone) [inline app/javascript/packs/application.js:11]: Very good 😆

**Review by TheGrayStone — CHANGES_REQUESTED** [review]
Looks good overall. I'm still missing some feature tests though... 😅 

How else will you know that the edit page really saves the user?

File: app/controllers/openingtimes_controller.rb [hunk 3]
```diff
@@ -48,10 +51,10 @@ def update
...
+    #     end
```
Comment (TheGrayStone) [inline app/controllers/openingtimes_controller.rb:57]:
```suggestion
```

**Review by TheGrayStone — CHANGES_REQUESTED** [review]
Oh, and please remove this 😅

**Review by Olliwehr — APPROVED** [review]
Overall it looks good to me! If this gets merged before the profile picture stuff, we probably need to adress some merge conflicts there. But thats okay for now :D

**Review by TheGrayStone — APPROVED** [review]

**Pull request merged**